### PR TITLE
[Dynamic Dashboard] Beta release fixes

### LINKED ISSUE #11523: Small fixes and improvements

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #11523 and addresses the issues found during beta testing.

**To test:**
1. Verify the dashboard screen menu buttons match the design (first share button
2. Verify the custom date range pencil icon matches the design (same as dashboard Edit menu button)
3. Verify that after trashing the only order, the dashboard cards related to orders are hidden

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/212">18.7</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/dashboard/data/ObserveStatsWidgetsStatus.kt [hunk 1]
```diff
@@ -44,14 +41,6 @@ class ObserveStatsWidgetsStatus @Inject constructor(
             }
         }
 
-    private suspend fun getHasOrdersFromOrderStatusOptions() = withContext(coroutineDispatchers.io) {
-        orderStore.getOrderStatusOptionsForSite(selectedSite.get())
-            .filter { it.statusKey != "checkout-draft" }
-            .takeIf { it.isNotEmpty() }
-            ?.sumOf { it.statusCount }
-            ?.let { it != 0 }
-    }
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/dashboard/data/ObserveStatsWidgetsStatus.kt]: @0nko as discussed internally, there is a way to fix this without losing the benefits of using the order status options, if you can please revert this change, and I'll create a separate PR that fixes it.

Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/dashboard/data/ObserveStatsWidgetsStatus.kt]: Sure, thanks!

**Review by hichamboushaba — APPROVED** [review]
Looks good, thanks @0nko

**Pull request merged**